app/routes/signup.js

```diff
@@ -7,17 +7,15 @@ export default class SignupRoute extends Route {
       credentials: 'include',
     });
     const userData = await response.json();
-
-    if (response.status === 401) {
+    if (response.status == 200 && !userData.incompleteUserDetails) {
+      alert("You already have filled the up form. You'll now be redirected.");
+      window.open('https://realdevsquad.com/goto', '_self');
+    } else {
       alert('You have not logged in. Please login first to fill this form.');
```

Comment:
Missing a very critical condition here: When the response is 200, the is having the cookie and the user has `incompleteUserDetails` set to true, This is THE condition.

According to current implementation, we're sending the user away when they're not logged in -- perfectly fine
We are also sending them away when they have the cookie and have not filled the form -- these are THE people who'll be using this feature, right?

Reply:
I'm not able to test only because of this 😅 
Conditions need to be modified @sumitd94

Reply:
Done!